validator/client/src/test/resources/tech/pegasys/teku/validator/client/restapi/paths/_eth_v1_validator_{pubkey}_feerecipient.json

```diff
@@ -33,6 +33,113 @@
           }
         }
       },
+      "401" : {
+        "description" : "Unauthorized, no token is found",
+        "content" : {
+          "application/json" : {
+            "schema" : {
+              "$ref" : "#/components/schemas/HttpErrorResponse"
+            }
+          }
+        }
+      },
+      "403" : {
+        "description" : "Forbidden, a token is found but is invalid",
+        "content" : {
+          "application/json" : {
+            "schema" : {
+              "$ref" : "#/components/schemas/HttpErrorResponse"
+            }
+          }
+        }
+      },
+      "404" : {
+        "description" : "Not found",
+        "content" : {
+          "application/json" : {
+            "schema" : {
+              "$ref" : "#/components/schemas/HttpErrorResponse"
+            }
+          }
+        }
+      },
+      "400" : {
+        "description" : "The request could not be processed, check the response for more information.",
+        "content" : {
+          "application/json" : {
+            "schema" : {
+              "$ref" : "#/components/schemas/HttpErrorResponse"
+            }
+          }
+        }
+      },
+      "500" : {
+        "description" : "Internal server error",
+        "content" : {
+          "application/json" : {
+            "schema" : {
+              "$ref" : "#/components/schemas/HttpErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "post" : {
+    "operationId" : "SetFeeRecipient",
+    "summary" : "Set validator fee recipient",
+    "description" : "Sets the validator client fee recipient mapping which will then update the beacon node. Existing mappings for the same validator public key will be overwritten. Configuration file settings will take precedence over this API, so if your validator fee recipient configuration file contains this public key, it will need to be removed before attempting to update with this api.Cannot specify default fee recipient address through API.\n\nWARNING: The fee_recipient is not used on Phase0 or Altair networks.",
```

Comment:
The "Cannot specify default fee recipient address through API" sentence has no space before it and I'm not really sure what it's trying to say - phrasing is a bit weird.

Reply:
copied from the api spec, can rephrase